src/janus-api/models/log-entry.js

```diff
@@ -21,10 +21,10 @@ export const createLogEntry = (type, content) => {
   that.muteRequestText = function() {
     var res;
 
-    if (that.content.source.isPublisher) {
```

Comment:
@imobachgs We should review this together, please.

I'm not sure if that change is really necessary.